.gitignore

```diff
@@ -31,14 +31,20 @@ test/dockerclients/.*
 # files generated by local dev,
 # do not need to check in and can be deleted
 # if not running local dev
-/local-dev/*.sock
+.npm
+conf/*.sock
+conf/__pycache__
+conf/jwtproxy_conf.yaml
+conf/mitm.cert
+conf/mitm.key
+conf/nginx/hosted-http-base.conf
+conf/nginx/http-base.conf
+conf/nginx/nginx.conf
+conf/nginx/rate-limiting.conf
+conf/nginx/server-base.conf
+conf/quay.kid
+conf/quay.pem
+conf/supervisord.conf
```

Comment:
Some of these files should not be ignored by git

Reply:
Can you be more specific?